### ISSUE TITLE
Compatibility table consistent versioning

### ISSUE THREAD

**Issue by scripthunter7** [AdguardTeam/AGLint#95]
I see that the "`version_added`" / "`version_removed`" field is used inconsistently in the compatibility table PRs.

For example:
- "Adblock Plus x.y"
- "x.y" / "x.y.z"
- "CoreLibs x.y.z"

Currently, the documentation does not go into detail on this (I just discovered this issue, sorry), but we should clarify these at the beginning and specify them consistently in the future.

At the moment I see this as a solution:
- Only the version should be specified without any text: "x.y" / "x.y.z"
  - For CoreLibs based apps, the version should be the CoreLibs version
  - Otherwise, the version of the adblocker (CB / extension)

Note: The current main idea is that the data is grouped by adblocker, so the fields are actually related to the adblocker (e.g. version/documentation). This makes the description redundant (write the same text 3 times), but that's the design at the moment.

Example:
```yml
adg_any:
  # Version is related to AG
  version: "x.y"

ubo_any:
  # Version is related to uBO
  version: "x.y.z"

any:
  # Version is related to any Adblocker (avoid this if possible)
  version: "x.y.z"
```

@Alex-302 @piquark6046 @zloyden @jellizaveta @AdamWr What do you think?
